lexers/x/xaml.go

```diff
@@ -0,0 +1,19 @@
+package x
+
+import (
+	. "github.com/alecthomas/chroma" // nolint
+	"github.com/alecthomas/chroma/lexers/internal"
+)
+
+// XAML lexer.
+var XAML = internal.Register(MustNewLexer(
```

Comment:
Didn't find it on pygments nor Chroma.

Reply:
This is a wakatime top language: https://wakatime.com/leaders/languages.